data-plane/core/src/main/java/dev/knative/eventing/kafka/broker/core/eventbus/ContractMessageCodec.java

```diff
@@ -22,26 +22,24 @@
 
 /**
  * This is a noop codec to send the contract on the event bus (works only in local environments)
- * <p>
- * https://github.com/eclipse-vertx/vert.x/issues/3375
+ *
+ * <p>https://github.com/eclipse-vertx/vert.x/issues/3375
```

Comment:
p should have its own line here

Reply:
google-java-style format code in this way, no way to change it by design.